src/plugins/vis_type_timeseries/public/timeseries_vis_renderer.tsx

```diff
@@ -39,6 +41,10 @@ export const getTimeseriesVisRenderer: (deps: {
   name: 'timeseries_vis',
   reuseDomNode: true,
   render: async (domNode, config, handlers) => {
+    // Build optimization. Move app styles from main bundle
+    // @ts-ignore
```

Comment:
Why does this require a `@ts-ignore`? Can we use `@ts-expect-error {put reason here}` instead?

Reply:
np 👍